trade-finance-manager-api/src/v1/helpers/amendment.helpers.test.js

```diff
@@ -21,10 +21,10 @@ const amendmentVariables = require('../__mocks__/amendmentVariables');
 const MOCK_NOTIFY_EMAIL_RESPONSE = require('../__mocks__/mock-notify-email-response');
 const MOCK_NOTIFY_EMAIL_BAD_RESPONSE = require('../__mocks__/mock-notify-email-bad-response');
 
+const mockUserInformation = { userType: 'test' };
```

Comment:
Note to self: this should be
```suggestion
const mockAuditDetails = { userType: 'test' };
```